app/views/layouts/_ae_resolve_options.html.haml

```diff
@@ -85,19 +85,6 @@
             :javascript
               miqInitSelectPicker();
               miqSelectPickerEvent('target_id', "#{url}")
-- if ae_sim_form
```

Comment:
No - it's fine there - Sometimes the user just wants to test but not run the method.  

For scheduling automate tasks the method will always run regardless if we set that parameter or not.